spec/game_spec.rb

```diff
@@ -0,0 +1,9 @@
+require_relative '../game'
+require_relative '../app'
+
+describe Game do
+  it 'id should be an integer' do
+    game = Game.new(1, '2019-01-01', true, true)
+    expect(game.id).to be_a(Integer)
+  end
+end
```

Comment:
- [ ] As stated in the requirements, all methods inside classes should be tested. So please ensure to add tests for the `can_be_archived?` methods from the `Game` class. 👌

### Requirement :point_down: 
![Screenshot from 2023-11-03 14-59-27](https://github.com/dvcorreia1984/ruby-group-capstone/assets/89228743/62dd0196-4c4f-4871-86a0-c8f997697f3b)